linux_os/guide/services/ssh/ssh_server/sshd_disable_compression/rule.yml

```diff
@@ -23,7 +23,6 @@ identifiers:
     cce@rhel8: CCE-80895-6
     cce@rhel9: CCE-90801-2
     cce@sle12: CCE-83062-0
-    cce@sle15: CCE-85647-6
```

Comment:
It is not a good idea to remove a `CCE` from a rule. In general, the once a rule receives a `cce`, they are tied forever.
https://complianceascode.readthedocs.io/en/latest/manual/developer/06_contributing_with_content.html#rules

Reply:
Thanks, @marcusburghardt ,

I've reverted the CCE changes.

Best regards and have a nice day!